Add a note about steps required for EL 7 in the upgrade steps for bra… (#1996)
theforeman/foreman-documentation: guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

baseurl=file:///media/sat6/Maintenance/
----

> . If the {Project} is running on a {RHEL} 7 system, configure the Ansible repository from the ISO file.

Reviewer: ```suggestion
. If your {Project} runs on {RHEL} 7, configure the Ansible repository from the ISO file.
```
